doc/api/deprecations.md

```diff
@@ -914,6 +914,14 @@ Type: Runtime
 
 This was never a documented feature.
 
+<a id="DEP0101"></a>
+### DEP0101: with-lttng
+
+Type: Documentation-only
+
+`with-lttng` has been an opt-in compile time option that was broken when trying to compile
```

Comment:
I think it would be better to just say the `with-lttng` compile time option is deprecated. If there is a recommended alternative, that could be listed here as well.

Reply:
Agreed